spec/services/vendor_api/single_application_presenter_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe VendorApi::SingleApplicationPresenter do
+  subject(:presenter) { described_class.new(application_choice) }
+
+  let(:application_choice) { create :application_choice }
+
+  around do |example|
+    Timecop.freeze do
+      example.run
```

Comment:
Nice! never seen this technique before

Reply:
Should we put this in the main spec helper? Seems useful.

Reply:
Feels a bit magic for the main helper, maybe?